app/reducers/companies.js

```diff
@@ -246,7 +250,8 @@ export const selectJoblistingsForCompany = createSelector(
     return joblistings.filter(
       joblisting =>
         joblisting.company &&
-        Number(joblisting.company.id) === Number(companyId)
+        Number(joblisting.company.id) === Number(companyId) &&
+        moment().isBefore(joblisting.deadline)
```

Comment:
Sometimes users want to see old joblistings, so that they can prepare for an interview. How can we handle this?